ql/src/java/org/apache/hadoop/hive/ql/exec/repl/bootstrap/load/LoadDatabase.java

```diff
@@ -21,9 +21,9 @@
 import org.apache.hadoop.hive.metastore.api.Database;
 import org.apache.hadoop.hive.metastore.api.InvalidOperationException;
 import org.apache.hadoop.hive.ql.ddl.DDLWork;
-import org.apache.hadoop.hive.ql.ddl.database.AlterDatabaseSetOwnerDesc;
-import org.apache.hadoop.hive.ql.ddl.database.AlterDatabaseSetPropertiesDesc;
-import org.apache.hadoop.hive.ql.ddl.database.CreateDatabaseDesc;
+import org.apache.hadoop.hive.ql.ddl.database.create.CreateDatabaseDesc;
+import org.apache.hadoop.hive.ql.ddl.database.setowner.AlterDatabaseSetOwnerDesc;
```

Comment:
setowner -> owner? or should we combine all set operations into a single one?

Reply:
The 3 set operations are all different, with different tokens (TOK_ALTERDATABASE_LOCATION, TOK_ALTERDATABASE_OWNER, TOK_ALTERDATABASE_PROPERTIES), and different parseable data, so the logic of the ddl requires them to be separate.  What I'd suggest is to have a "org.apache.hadoop.hive.ql.ddl.database.alter" package, and under that a separate "location", "owner", and "properties" package for the 3 types.

Please let me know what you think.

Reply:
Yes, your proposal makes sense, let's have 3 subpackages.

Reply:
Created 3 sub packages under alter.